.github/workflows/CI.yml

```diff
@@ -375,12 +375,16 @@ jobs:
 
       - run: ls -R
 
+        # Build the ssm package and manifest by using a version with github short sha as same as e2etest-preparationfor the following reasons:
+        # - Distingush each build for Integration test
+        # - Increase more visibility to the customer and also for the dev since we publish the image for integration test.
+        # - Avoid having similar layers image and only have a final result image.
```

Comment:
These reasons don't really apply for the SSM package. I don't think we give customers access and it isn't an image.

Reply:
True thought. Reverted to previous comment but add some extra things.